AboutPage.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Me</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<hr>
+<body>
+        <ul>
+                <a href="AboutPage.html" target="_blank">About Me</a>
+                <a href="Portfolio.html" target="_blank">Portfolio</a>
+                    </ul>
+                    <hr>
+       
+    <h3>Experience</h3>
+    <p>
+            
+
+            I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an <b><em>awesome</em></b> Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
+            
+            
+     <h4>Relevant Experience</h4>  
+          <div class="Relaxr">
+            <strong>Web Developer</strong> , Relaxr 2015 - present
+            Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+            <img src="/Users/mesfernasser/sei/homeworks/hw-w02d01-website/starter-code/images/relaxr-deliverable.png" alt="Relaxr" controls>
```

Comment:
Use relative hyperlinks instead of absolute so the images can be properly displayed on other users devices. For example: src="starter-code/images/relaxr-deliverable.png"